### ISSUE TITLE
Stacks to support

### ISSUE THREAD

**Issue by housseindjirdeh** [GoogleChrome/lighthouse-stack-packs#7]
Creating this issue to consolidate a master list of stacks/platforms to consider supporting moving forward. This is by no means final and can change at anytime, but it's probably helpful to have a rough list :).

- CMS/E-commerce
  - WordPress
  - Magento
  - Drupal
  - Shopify
  - Joomla
  - TYPO3
- JS frameworks/libraries
  - Angular
  - React
  - Vue

If you have any suggestions for adding/removing any stack on this list, please don't hesitate to let us know.

**Comment by machour** [comment]
Adding some more PHP stacks to consider:

**Frameworks:**
 * [Laravel](http://laravel.com)
 * [Symfony](https://symfony.com/)
 * [Yii](http://yiiframework.com)
 * [Slim](http://www.slimframework.com/)
 * [Cake PHP](https://cakephp.org/)

**Site builders:**
 * [SPIP](https://www.spip.net)

as these are PHP frameworks, it can get tricky to identify them as they usually don't dictate anything for layout and users are free to implement whatever they want.

Some may provide some core JS component, that may or *may not* be present (like yii.js for a Yii website), so it could be an additional hint.

Feel free to ping me for PHP related questions, I'll be more than glad to contribute something back to LightHouse ❤️ 

*PS: in PHP development, leaking some information about the stack used, like the PHP version number, is considered a bad practice as attacker can exploit known issues on old versions. It would be great if Lighthouse could hint the user about that, but I'm not sure if that fall into LightHouse scope.*

**Comment by housseindjirdeh** [comment]
This is great, thanks a ton for this @machour. Would definitely be amazing to see PHP tooling supported with stack packs.

Do you think we would get sufficient interest from these communities to help us provide audit suggestions for these stacks? We would love to have as many stacks supported as we eventually can, but we also need to try to ensure that we have community members on board that drive the write-up of these audit suggestions. (I know you can help significantly here, but the more help the merrier :))

**Comment by dwsmart** [comment]
In addition to React & Vue, support for popular higher-level frameworks like next & nuxt could be great. 

Should be fairly easy to detect by looking for includes from _nuxt / _next etc.

**Comment by housseindjirdeh** [comment]
@dwsmart Agreed, [Next](https://github.com/johnmichel/Library-Detector-for-Chrome/blob/master/library/libraries.js#L391) and [Nuxt](https://github.com/johnmichel/Library-Detector-for-Chrome/blob/master/library/libraries.js#L1133) detections are already in place :)

I think the question here would be trying to figure out different, appropriate stack-suggestions that can stay separate between React + Next or Vue + Nuxt. Definitely worth exploring however!

**Comment by wardpeet** [comment]
We would love to have gatsby in here as well

cc @KyleAMathews
